eva/udfs/abstract/hf_abstract_udf.py

```diff
@@ -80,11 +80,14 @@ def output_formatter(self, outputs: Any):
             return pd.DataFrame(outputs, index=[0])
         # PERF: Can improve performance by avoiding redundant list creation
         result_list = []
-        for row_output in outputs:
-            # account for the case where we have more than one prediction for an input
-            if isinstance(row_output, list):
-                row_output = {k: [dic[k] for dic in row_output] for k in row_output[0]}
-            result_list.append(row_output)
+        if outputs != [[]]:
+            for row_output in outputs:
+                # account for the case where we have more than one prediction for an input
+                if isinstance(row_output, list):
+                    row_output = {k: [dic[k] for dic in row_output] for k in row_output[0]}
+                result_list.append(row_output)
+        elif self.pipeline_args["task"] == "ner":
+            result_list.append({"entity": "", "score": 0, 'index': 0, 'end': 0, 'word': 0, 'start': 0})
```

Comment:
Why are we hardcoding this here? The above code should take care of it automatically right? Is there a case when it doesn't work?

Reply:
it is not taking care of it automatically , it is giving error of unknown column found as shown in below image 

<img width="496" alt="image" src="https://github.com/georgia-tech-db/eva/assets/42251896/9e53558d-075a-48a5-8457-64ab20c561b4">

Reply:
@affan00733 Why is there an unknown column coming up with only the NER task? What happens with some other task like text classification? Why does this code not work: 

`row_output = {k: [dic[k] for dic in row_output] for k in row_output[0]}`

We should avoid specializing the code for a particular task as much as possible

Reply:
For the same text classification it is working properly , none of the issues have been encountered